lib/modules/platform/github/massage-markdown-links.ts

```diff
@@ -58,12 +68,18 @@ export function massageMarkdownLinks(content: string): string {
   try {
     const rightSpaces = content.replace(content.trimEnd(), '');
     const matches: UrlMatch[] = [];
-    remark().use(collectLinkPosition(content, matches)).processSync(content);
+    const newContent = content.replace(
+      regEx(/@&#8203;(?<name>[a-zA-Z1-9-!$%^&*()_+|~=`{}[\]:";'<>?,.]*)/gi),
```

Comment:
same here, i don't think all those special chars are needed, as they aren't allowed by GitHub

Reply:
you are right 
the allowed chars are `_-.`